app/README.md

```diff
@@ -0,0 +1,35 @@
+# App
+
+A simple FE application for showing different cities and their pictues.
```

Comment:
Just a small nice to have type of comment here - might as well add the "editing" part to the general description (so it would be something like this - "A simple FE application for showing and editing different cities and their pictues."

Reply:
Done!